dbms/src/Flash/tests/gtest_join_executor.cpp

```diff
@@ -738,10 +768,11 @@ try
     ColumnsWithTypeAndName left_column_data;
     ColumnsWithTypeAndName right_column_data;
     ColumnsWithTypeAndName common_column_data;
-    size_t table_rows = 102400;
-    size_t common_rows = 20480;
+    size_t table_rows = 51200;
```

Comment:
why change this

Reply:
> why change this

The tests are running too slowly，to prevent CI execution timeouts, halve the amount of data.